src/core/stdc/config.d

```diff
@@ -104,31 +104,28 @@ version( Windows )
     enum __c_long  : int;
     enum __c_ulong : uint;
 
-    alias __c_long   cpp_long;
-    alias __c_ulong  cpp_ulong;
-
     alias int   c_long;
     alias uint  c_ulong;
 
+    alias __c_long   cpp_long;
+    alias __c_ulong  cpp_ulong;
+
     alias long  cpp_longlong;
     alias ulong cpp_ulonglong;
 }
 else version( Posix )
 {
   static if( (void*).sizeof > int.sizeof )
   {
-    enum __c_long  : long;
-    enum __c_ulong : ulong;
-
     enum __c_longlong  : long;
     enum __c_ulonglong : ulong;
 
-    alias __c_long   cpp_long;
-    alias __c_ulong  cpp_ulong;
-
     alias long  c_long;
     alias ulong c_ulong;
 
+    alias long   cpp_long;
+    alias ulong  cpp_ulong;
+
```

Comment:
Are `__c_long` and `__c_ulong` used anywhere else? These are no longer defined, so any usage will break.

Reply:
Nope, and they aren't supposed to be. `__c_longlong` is also only defined where needed.